holoviews/plotting/mpl/util.py

```diff
@@ -131,25 +130,24 @@ def validate(style, value, vectorized=True):
 
 
 def filter_styles(style, group, other_groups, blacklist=None):
-    """
-    Filters styles which are specific to a particular artist, e.g.
+    """Filters styles which are specific to a particular artist, e.g.
     for a GraphPlot this will filter options specific to the nodes and
     edges.
 
-    Arguments
-    ---------
-    style: dict
+    Parameters
+    ----------
+    style : dict
         Dictionary of styles and values
-    group: str
+    group : str
         Group within the styles to filter for
-    other_groups: list
+    other_groups : list
         Other groups to filter out
-    blacklist: list (optional)
+    blacklist : list, optional
```

Comment:
Is this how you say optional?

Reply:
Yes. See https://numpydoc.readthedocs.io/en/latest/format.html#parameters